fix: 显示问题

heyumoAi opened the pull request:
Description: cat ppt文件，Tab会导致陷入死循环

Log: bug https://pms.uniontech.com/bug-view-164711.html

BLumia reviewed (commented):
Not quite sure but if we need to support distro with older vala version, maybe it's better to add a [vala api version check (something like `VALA_X_Y`)](https://wiki.gnome.org/Projects/Vala/Manual/Preprocessor)?

felixonmars commented:
> Not quite sure but if we need to support distro with older vala version, maybe it's better to add a [vala api version check (something like `VALA_X_Y`)](https://wiki.gnome.org/Projects/Vala/Manual/Preprocessor)?

Oops, actually I meant VTE 0.60... I'll try to see if there is a way to check for that.

felixonmars commented:
Fixed now :)

Commit:
fix: 显示问题

Description: cat ppt文件，Tab会导致陷入死循环

Log: bug https://pms.uniontech.com/bug-view-164711.html

hundundadi commented:
[是否满足兼容性要求] Y
[是否满足commit提交规范] Y
[是否满足编码规范] Y
[Review结论] Pass
[Fail原因] N/A

pengfeixx reviewed (approved):
/+1
